.github/workflows/CICD.yml

```diff
@@ -105,6 +100,18 @@ jobs:
           arm-unknown-linux-gnueabihf) sudo apt-get -y update ; sudo apt-get -y install gcc-arm-linux-gnueabihf ;;
           aarch64-unknown-linux-gnu) sudo apt-get -y update ; sudo apt-get -y install gcc-aarch64-linux-gnu ;;
         esac
+    - name: Install 'cargo get'
+      uses: actions-rs/install@v0.1
+      with:
+        crate: cargo-get
+        version: latest
```

Comment:
Maybe that should be set to a fixed version.

Reply:
I think so, yes (but I won't be upset if you keep it as is).